Include/internal/pycore_ceval.h

```diff
@@ -25,8 +25,10 @@ PyAPI_FUNC(int) _PyEval_AddPendingCall(
     int (*func)(void *),
     void *arg);
 PyAPI_FUNC(void) _PyEval_SignalAsyncExc(PyThreadState *tstate);
+#ifdef HAVE_FORK
 PyAPI_FUNC(void) _PyEval_ReInitThreads(
     struct pyruntimestate *runtime);
```

Comment:
I don't think that this function must be exported. Please make it private with:

```suggestion
extern void _PyEval_ReInitThreads(struct pyruntimestate *runtime);
```